README.md

```diff
@@ -71,6 +71,20 @@ ClusterServiceVersions can be collected into `CatalogSource`s which will allow a
 
 Learn more about the components used by OLM by reading about the [architecture] and [philosophy].
 
+# Overall OLM Chart
+
+![OLM](./olm.png)
+
+From the above chart, we can see that there are 2 steps required for a new operator onboard:
```

Comment:
I would expect this section to describe the diagram, rather than explain how to install an operator with OLM. There are/should be docs about that (try looking at the OLM book).